cmd/mimir/help-all.txt.tmpl

```diff
@@ -1029,6 +1031,8 @@ Usage of ./cmd/mimir/mimir:
     	Max series that this ingester can hold (across all tenants). Requests to create additional series will be rejected. 0 = unlimited.
   -ingester.instance-limits.max-tenants int
     	Max tenants that this ingester can hold. Requests from additional tenants will be rejected. 0 = unlimited.
+  -ingester.max-global-ephemeral-series-per-user int
+    	[experimental] The maximum number of in-memory ephemeral series per tenant, across the cluster before replication. 0 to disable. (default 150000)
```

Comment:
I think `0 to disable` sounds like the limit will be disabled, which would mean that the number of ephemeral series per user would then be unlimited. But looking at the code, this actually disables the ingestion of ephemeral samples, so I think maybe it would be better to say something more explicit like `0 to disable ephemeral storage`.

Reply:
That's a good point. In many limits we treat 0 as unlimited, but in others (eg. exemplars, out-of-order ingestion) we use 0 to disable the feature. I will make this help text more explicit about what it does.

Reply:
Updated in https://github.com/grafana/mimir/pull/4004/commits/e2006bb5cf3f03bd0423af3630b4d301e8f16285

Reply:
(Docs regenerated in https://github.com/grafana/mimir/pull/4004/commits/31203176c196cfa31dff0debc14a695c9a12707f)